run_presubmit.sh

```diff
@@ -22,16 +22,7 @@ if [[ -z `which coverage` ]];then
   echo "coverage is not installed. Installing ..."
   python -m pip install coverage
 fi
-# coverage run --source=gcp_variant_transforms setup.py test
-# Add individual files and packages as they are migrated to Python 3. Once all
-# of the files are migrated, delete this code and uncomment the above line.
-coverage run --source=gcp_variant_transforms setup.py test -s setup.py
-coverage run --source=gcp_variant_transforms setup.py test -s \
-  gcp_variant_transforms.beam_io
-coverage run --source=gcp_variant_transforms setup.py test -s \
-  gcp_variant_transforms.libs.annotation
-coverage run --source=gcp_variant_transforms setup.py test -s \
-  gcp_variant_transforms.libs.variant_merge
```

Comment:
I am guessing these were removed because integration tests were not passing back then.

Do you think there is a better way to do this?

I am afraid we make some changes here and we never recover them.

Reply:
No, this is not for integration tests. After migrating all subfolder unit tests, we restore to run coverage over all of the  unit tests (instead of specific subfolders).